server/opts.go

```diff
@@ -3444,6 +3460,24 @@ func parseTLS(v interface{}, isClientCtx bool) (t *TLSConfigOpts, retErr error)
 				at = mv
 			}
 			tc.Timeout = at
+		case "pinned_certs":
+			ra, ok := mv.([]interface{})
+			if !ok {
+				return nil, &configErr{tk, "error parsing tls config, expected 'pinned_certs' to be a list of sha256(DER encoded SubjectPublicKeyInfo)"}
```

Comment:
This could usefully state hex-encoded no-colons

Reply:
fixed